scipy/stats/tests/test_multivariate.py

```diff
@@ -1635,6 +1637,180 @@ def test_haar(self):
         res = kstest(x.ravel(), uniform(-np.pi, 2*np.pi).cdf)
         assert_(res.pvalue > 0.05)
 
+
+class TestMultivariateHypergeom(object):
+    def test_logpmf(self):
+        vals1 = multivariate_hypergeom.logpmf(x=[3, 4], m=[5, 10], n=7)
+        assert_allclose(vals1, -1.1198144963187122, rtol=1e-8)
```

Comment:
Tests look pretty good, but where do hard-coded ground truth values come from?

Reply:
These values have been generated from this distribution only. I still need to add a test comparing with R/mathematica. I will add that after addressing other reviews, if it is fine with you.

Reply:
That's fine, but instead of adding a separate test, just replace these values with ones from R, Mathematica, or somewhere else. I think it's somewhat redundant to be `allclose` to both a ground truth (e.g. R, Mathematica) AND `allclose` to former output. I could understand the value of being `allclose` to a ground truth and `equal` to a former output, but that is messy across platforms.

Reply:
This has finally been addressed in https://github.com/scipy/scipy/pull/12839/commits/f94a09df0ef290dd7b6160e19d66f28b9e56603d